demos/list.html

```diff
@@ -53,7 +53,7 @@
               info
             </a>
           </li>
-          <li class="mdc-list-item">
+          <li class="mdc-list-item mdc-list-item--activated">
```

Comment:
It would be helpful to have separate demos for following cases:

- singleSelect: with `--selected`
- singleSelect: with `--activated`
- multiSelect

Reply:
I'll add a demo for `--activated`.